src/components/App.js

```diff
@@ -1,20 +1,38 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
-import Menu from '../containers/MenuContainer.js';
-import Cat from './Cat.js';
-import Message from './Message.js';
-import CallToAction from './CallToAction.js';
-import '../styles/App.css';
+import Menu from "../containers/MenuContainer.js";
+import Cat from "./Cat.js";
+import Message from "./Message.js";
+import CallToAction from "./CallToAction.js";
+import Ball from "./Ball.js";
+import "../styles/App.css";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isBouncing: false
+    }
+    this.makeBounce = this.makeBounce.bind(this);
+  }
+  makeBounce() {
+    /* HACK ATTACK */
+    if(this.state.isBouncing) { return; }
+    this.setState({ isBouncing: true });
+    window.setTimeout(() => {
```

Comment:
Commit [985c60d](https://github.com/CYPIAPT-LNDSE/friendly-friends/pull/29/commits/985c60d8016c8789e3c1a73a2befc5d1e6654af6) should fix that.

Out of interest, how would we end up in a situation where the component unmounts during the animation?